fml/meta/fml_tests.cmx

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If this is going to be common for all the tests, say `shell_tests` as well in the future. Would it make sense for us to place it at a more generic location and caling it `fucshia_tests.cmx` or something?